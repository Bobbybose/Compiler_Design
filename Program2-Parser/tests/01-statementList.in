a = 7;
a = 2*2; print(b); if(c==d) e=3;
