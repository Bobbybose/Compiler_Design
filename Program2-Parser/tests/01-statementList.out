statementList: 1,1
|   
+-- statementList (EMPTY STRING): 1,1
|   
+-- statement: 1,1
|   |   
|   +-- expression: 1,1
|   |   |   
|   |   +-- expression: 1,1
|   |   |   |   
|   |   |   +-- a: 1,1
|   |   |   |   
|   |   |   *   
|   |   |   
|   |   +-- =: 1,3
|   |   |   
|   |   +-- expression: 1,5
|   |   |   |   
|   |   |   +-- 7: 1,5
|   |   |   |   
|   |   |   *   
|   |   |   
|   |   *   
|   |   
|   +-- ;: 1,6
|   |   
|   *   
|   
*   
statementList: 2,1
|   
+-- statementList: 2,1
|   |   
|   +-- statementList: 2,1
|   |   |   
|   |   +-- statementList (EMPTY STRING): 2,1
|   |   |   
|   |   +-- statement: 2,1
|   |   |   |   
|   |   |   +-- expression: 2,1
|   |   |   |   |   
|   |   |   |   +-- expression: 2,1
|   |   |   |   |   |   
|   |   |   |   |   +-- a: 2,1
|   |   |   |   |   |   
|   |   |   |   |   *   
|   |   |   |   |   
|   |   |   |   +-- =: 2,3
|   |   |   |   |   
|   |   |   |   +-- expression: 2,5
|   |   |   |   |   |   
|   |   |   |   |   +-- expression: 2,5
|   |   |   |   |   |   |   
|   |   |   |   |   |   +-- 2: 2,5
|   |   |   |   |   |   |   
|   |   |   |   |   |   *   
|   |   |   |   |   |   
|   |   |   |   |   +-- *: 2,6
|   |   |   |   |   |   
|   |   |   |   |   +-- expression: 2,7
|   |   |   |   |   |   |   
|   |   |   |   |   |   +-- 2: 2,7
|   |   |   |   |   |   |   
|   |   |   |   |   |   *   
|   |   |   |   |   |   
|   |   |   |   |   *   
|   |   |   |   |   
|   |   |   |   *   
|   |   |   |   
|   |   |   +-- ;: 2,8
|   |   |   |   
|   |   |   *   
|   |   |   
|   |   *   
|   |   
|   +-- statement: 2,10
|   |   |   
|   |   +-- expression: 2,10
|   |   |   |   
|   |   |   +-- function_call: 2,10
|   |   |   |   |   
|   |   |   |   +-- print: 2,10
|   |   |   |   |   
|   |   |   |   +-- (: 2,15
|   |   |   |   |   
|   |   |   |   +-- function_args: 2,16
|   |   |   |   |   |   
|   |   |   |   |   +-- expression: 2,16
|   |   |   |   |   |   |   
|   |   |   |   |   |   +-- b: 2,16
|   |   |   |   |   |   |   
|   |   |   |   |   |   *   
|   |   |   |   |   |   
|   |   |   |   |   *   
|   |   |   |   |   
|   |   |   |   +-- ): 2,17
|   |   |   |   |   
|   |   |   |   *   
|   |   |   |   
|   |   |   *   
|   |   |   
|   |   +-- ;: 2,18
|   |   |   
|   |   *   
|   |   
|   *   
|   
+-- statement: 2,20
|   |   
|   +-- if: 2,20
|   |   
|   +-- (: 2,22
|   |   
|   +-- expression: 2,23
|   |   |   
|   |   +-- expression: 2,23
|   |   |   |   
|   |   |   +-- c: 2,23
|   |   |   |   
|   |   |   *   
|   |   |   
|   |   +-- ==: 2,24
|   |   |   
|   |   +-- expression: 2,26
|   |   |   |   
|   |   |   +-- d: 2,26
|   |   |   |   
|   |   |   *   
|   |   |   
|   |   *   
|   |   
|   +-- ): 2,27
|   |   
|   +-- statement: 2,29
|   |   |   
|   |   +-- expression: 2,29
|   |   |   |   
|   |   |   +-- expression: 2,29
|   |   |   |   |   
|   |   |   |   +-- e: 2,29
|   |   |   |   |   
|   |   |   |   *   
|   |   |   |   
|   |   |   +-- =: 2,30
|   |   |   |   
|   |   |   +-- expression: 2,31
|   |   |   |   |   
|   |   |   |   +-- 3: 2,31
|   |   |   |   |   
|   |   |   |   *   
|   |   |   |   
|   |   |   *   
|   |   |   
|   |   +-- ;: 2,32
|   |   |   
|   |   *   
|   |   
|   *   
|   
*   
